editor/src/components/editor/editor-component.tsx

```diff
@@ -516,7 +519,7 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
                   overflowX: 'hidden',
                 }}
               >
-                <DesignPanelRoot />
+                <DesignPanelRoot animationScope={props.animationScope} />
```

Comment:
why the prop drilling here? couldn't / shouldn't this be in a context? is props.animationScope referentially stable?

Reply:
what about this https://github.com/concrete-utopia/utopia/pull/6046/commits/b60e41e87183dca88ebe0e6f513178a4fc01fbef ?